Makefile

```
@@ -3,6 +3,8 @@ AMARU_PEER_ADDRESS ?= 127.0.0.1:3000
 HASKELL_NODE_CONFIG_DIR ?= cardano-node-config
 DEMO_TARGET_EPOCH ?= 173
 HASKELL_NODE_CONFIG_SOURCE := https://book.world.dev.cardano.org/environments
+COVERAGE_DIR ?= coverage
+COVERAGE_CRATES ?=
 
 .PHONY: help bootstrap run import-snapshots import-headers import-nonces download-haskell-config
```

```suggestion
.PHONY: help bootstrap run import-snapshots import-headers import-nonces download-haskell-config coverage-html coverage-lconv check-llvm-cov
```

Either this, or change the `coverage-lconv` rule to: 

```
lcov.info: 
    ....
```